network/roles/google_cloud_vpc/tasks/absent.yaml

```diff
@@ -0,0 +1,45 @@
+---
+- name: remove google cloud vpc firewall rules
+  google.cloud.gcp_compute_firewall:
+    name: "{{ item.name }}"
+    allowed: "{{ item.allowed }}"
+    source_ranges: "{{ item.source_ranges | default(omit) }}"
+    network: { selfLink: "{{ gcp_network.selfLink }}" }
```

Comment:
Curious, how is `gcp_network` defined here?  I'm guessing it's external, just trying to understand the flow.